tests/test_convolution.cpp

```diff
@@ -257,5 +259,5 @@ static int test_convolution_1()
 int main()
 {
     SRAND(7767517);
-    return test_convolution_0() || test_convolution_1() || test_convolution_2();
+    return test_convolution_2() || test_convolution_1() || test_convolution_0();
```

Comment:
order reversed  :smile:

Reply:
I know but otherwise it fails🤭

Reply:
I suspect some alllocator issue, I will investigate more :) It was weird as it didn't fail on the ctest coverage I added but only a couple of subtests later. And it is fixed as I said by switching the order around 🤷‍♂️

Reply:
Undid the re-order and actually fixed the issue :)